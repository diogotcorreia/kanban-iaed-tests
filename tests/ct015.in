a DJJDSJAHBDHJC
a 344JDHDH DSJSH
a |""# DJSF 473 F
a SD<334%%&/ SDHDH
a WSSFEHSH SAKSJJS
a "#$%&//%$(()??)
a ^SSHSH 2|!224 S
a
q
