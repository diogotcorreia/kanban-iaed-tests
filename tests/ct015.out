TO DO
IN PROGRESS
DONE
DJJDSJAHBDHJC
344JDHDH DSJSH
|""# DJSF 473 F
SD<334%%&/ SDHDH
WSSFEHSH SAKSJJS
"#$%&//%$(()??)
^SSHSH 2|!224 S
